users/migrations/0027_set_username_to_email.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+
+def update_usernames_to_email(apps, schema_editor):
+    """Update usernames to match user email addresses"""
+    User = apps.get_model("users", "User")
+
+    # Update usernames to match emails but truncate at max length
+    # Get max_length from username field
+    max_length = User._meta.get_field("username").max_length  # noqa: SLF001
+
+    users = User.objects.all()
+    for user in users:
```

Comment:
This will load all the users into memory which will have a chance of crashing in production with the large user count. Better run a bulk `update()` call over medium-sized slices of the user table.